docs/howto/system_testing.md

```diff
@@ -402,6 +410,15 @@ for system tests.
 
 | Option | Type | Required | Description |
 |---|---|---|---|
+| agent.linux_capabilities | array string | | Linux Capabilities that must been enabled in the system to run the Elastic Agent process. |
+| agent.pid_mode | string | | Turns on sharing between container and the host operating system the PID address space. |
+| agent.ports | array string | | List of ports to be exposed to access to the Elastic Agent.|
+| agent.runtime | string | | Runtime to run Elastic Agent process. |
+| agent.pre_start_script.language | string | | Programming language of the provisioning script. Default: `bash`.|
```

Comment:
Maybe we should also default it to `sh`, given that it is what is used in the provisioning script?

Reply:
I mixed the default languages for both scripts.
In the pre-start script, it must be `sh`. This is the only supported for now.
For the provisioning script, there is no restriction and the default one is bash.

I'll update those default values.

Reply:
Could we make both to be `sh` by default? It looks weird to have different defaults for them.

Reply:
Yes, it could be done like that, no problem.
I'll update elastic-package and package-spec :+1:

Reply:
Indeed, in code it was set for both scripts `sh` as default language:
- https://github.com/elastic/elastic-package/blob/7b156a51b8db9f300d4ba2fb67d6bcea9b744b42/internal/testrunner/runners/system/test_config.go#L113
- https://github.com/elastic/elastic-package/blob/7b156a51b8db9f300d4ba2fb67d6bcea9b744b42/internal/agentdeployer/info.go#L16